### PR TITLE
Complete service test cases with diagnostics

### PR DESCRIPTION
**Pull request by dilanSachi**
## Purpose
$subject

### CODE REVIEW

File: openapi-cli/src/test/java/io/ballerina/openapi/generators/service/ServiceDiagnosticTests.java [hunk 1]
```diff
@@ -193,7 +193,7 @@ public Object[][] fileProviderForOpenAPI31DiagnosticCheck() {
                 {"3.1.0_openapis/listennotes.yaml"},
                 {"3.1.0_openapis/placekit.yaml"},
                 {"3.1.0_openapis/urlbox.yaml"},
-//                {"3.1.0_openapis/vercel.yaml"}, // uncomment when /openapi-tools/issues/1332 is fixed
+                {"3.1.0_openapis/vercel.yaml"}, // uncomment when /openapi-tools/issues/1332 is fixed
```
Comment (NipunaRanasinghe) [inline openapi-cli/src/test/java/io/ballerina/openapi/generators/service/ServiceDiagnosticTests.java:196]:
```suggestion
                {"3.1.0_openapis/vercel.yaml"},
```

File: openapi-core/src/main/java/io/ballerina/openapi/core/service/ServiceGenerationHandler.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
+package io.ballerina.openapi.core.service;
```
Comment (NipunaRanasinghe) [inline openapi-core/src/main/java/io/ballerina/openapi/core/service/ServiceGenerationHandler.java:1]: missing license header

File: openapi-core/src/main/java/io/ballerina/openapi/core/service/ServiceGenerationHandler.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+import org.ballerinalang.formatter.core.Formatter;
+import org.ballerinalang.formatter.core.FormatterException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static io.ballerina.openapi.core.generators.common.GeneratorConstants.DEFAULT_FILE_HEADER;
+import static io.ballerina.openapi.core.generators.common.GeneratorConstants.DO_NOT_MODIFY_FILE_HEADER;
+
+public class ServiceGenerationHandler {
```
Comment (NipunaRanasinghe) [inline openapi-core/src/main/java/io/ballerina/openapi/core/service/ServiceGenerationHandler.java:17]: missing java docs

File: openapi-core/src/main/java/io/ballerina/openapi/core/service/ServiceGenerationHandler.java (hunk 2 above)
Comment (dilanSachi) [reply openapi-core/src/main/java/io/ballerina/openapi/core/service/ServiceGenerationHandler.java:1]: This is a  temporary branch that we have to setup for easy collaboration. Once all files are correctly finalised, we will go through all and fix the docs, licenses etc. Hope that is fine

**Pull request merged**